.gitignore

```diff
@@ -2,3 +2,5 @@ _output/
 _artifacts/
 lima.REJECTED.yaml
 .config
+.idea/
+.DS_Store
```

Comment:
These patterns are not specific to the Lima project and therefore don't belong into the repo.

>Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

(from https://git-scm.com/docs/gitignore)